pvlib/pvsystem.py

```diff
@@ -536,7 +536,7 @@ def pvsyst_celltemp(self, poa_global, wind_speed, temp_air):
         kwargs = _build_kwargs(['eta_m', 'alpha_absorption'],
                                self.module_parameters)
         return pvsyst_celltemp(poa_global, wind_speed, temp_air,
```

Comment:
Need to update argument order here.  Change argument order of `PVSystem.pvsyst_celltemp` to match the underlying function's order of irradiance, temperature, windspeed